Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -53,6 +53,25 @@ open ComplexConjugate
 
 /--
 This typeclass captures properties shared by ℝ and ℂ, with an API that closely matches that of ℂ.
+
+While the name of the class begins with `Is`,
+this typeclass carries data (all operations common to real and complex numbers),
```

Comment:
The data which annoys me (because a large chunk of the theorems about IsROrC do not need it, and we are going to pay for this later) is the choice of sqrt(-1); maybe this should be mentioned?

Reply:
I added a paragraph about this. You may add a paragraph about why it's bad (what price are we going to pay for this), e.g., in the not-yet-written `### TODO` section of the module docstring.

Reply:
I added a paragraph about AlgebraicClosure(Real) in the module docstring.